### ISSUE TITLE
Move Geb website to a static website. 

### ISSUE THREAD

**Issue by sdelamo** [apache/groovy-geb#208]
see: https://infra.apache.org/project-site.html

**Comment by jonnybot0** [comment]
@sdelamo - just wanted to check in with you on this. If I recall correctly, our basic plan is to set up a separate repository (groovy-geb-site, per the ASF conventions).

> # HTML Files
> You can use any other tool that generates static HTML pages, or hand-code those pages. You then check them into your project's website repository. The check-in will trigger a site update.

I think the first step would be to request the repository via the self serve portal at https://selfserve.apache.org/.

From there, we can setup a CI build or a Github action to build from the doc module and publish to that repository.

Does that sound about right? If you need, I can fill out the self service request for the repo. Just @me. :)

**Comment by sdelamo** [comment]
@jonnybot0 give a couple of weeks and I will start working on this.

**Comment by paulk-asert** [comment]
This is currently in the geb directory in the groovy-dev-site repo. So, I think we can close this.

**Comment by jonnybot0** [comment]
Agreed. I've got an open ticket with the ASF infra team to take over the domain. https://issues.apache.org/jira/browse/INFRA-26562

**Comment by jonnybot0** [comment]
Redirects are up and working from the old domain to the new one.


* gebish.org/manual/5.1/#using-pages
* http://www.gebish.org/manual/5.1/#using-pages
* https://www.gebish.org/manual/6.0/api/
* https://gebish.org/manual/6.0/api/
